cmd/deploy/remote.go

```diff
@@ -140,14 +141,15 @@ func (rd *remoteDeployCommand) deploy(ctx context.Context, deployOptions *Option
 
 	buildOptions := build.OptsFromBuildInfo("", "", buildInfo, &types.BuildOptions{Path: cwd, OutputMode: "deploy"})
 	buildOptions.Tag = ""
+	buildOptions.Manifest = deployOptions.Manifest
 
 	// we need to call Build() method using a remote builder. This Builder will have
 	// the same behavior as the V1 builder but with a different output taking into
 	// account that we must not confuse the user with build messages since this logic is
 	// executed in the deploy command.
 	if err := rd.builderV1.Build(ctx, buildOptions); err != nil {
 		return oktetoErrors.UserError{
-			E: fmt.Errorf("Error during development environment deployment."),
+			E: errors.Wrap(err, "Error during development environment deployment"),
```

Comment:
[github.com/pkg/errors](https://github.com/pkg/errors) is not part of the standard library and was replaced with the extended notation in [Go 1.13.](https://go.dev/blog/go1.13-errors)
```suggestion
			E: fmt.Errorf("Error during development environment deployment: %w", err),
```

Reply:
thanks! Updated ✅